fdbserver/Knobs.cpp

```diff
@@ -401,6 +401,7 @@ void ServerKnobs::initialize(bool randomize, ClientKnobs* clientKnobs, bool isSi
 	init( RESET_RESOLVER_BATCHES,                                 200 );
 	init( RESET_MASTER_DELAY,                                   300.0 );
 	init( RESET_RESOLVER_DELAY,                                 300.0 );
+	init( DECAY_TXN_RATE,                                        true ); if( randomize && BUGGIFY ) { DECAY_TXN_RATE = deterministicRandom()->random01() < 0.5 ? true : false; }
```

Comment:
This probably doesn't matter much, but we could optionally make this a little more specific. For example: `DECAY_TXN_RATE_ON_LEASE_EXPIRATION`

Reply:
Done.